static/src/stylesheets/module/content/live-blog/navigation.scss

```diff
@@ -42,82 +39,103 @@
 
 .liveblog-navigation__link {
     display: inline-block;
+    position: relative;
+    color: colour(neutral-1);
+    background: transparent;
+    border: 1px solid colour(neutral-4);
+    height: $gs-baseline * 3;
+    line-height: 38px;
+    font-weight: bold;
+    vertical-align: middle;
 
-    // Override user-agent styles for <a>
     &:hover {
+        border-color: colour(neutral-2);
         text-decoration: none;
     }
 
-    .inline-icon {
-        vertical-align: middle;
-        display: inline-block;
+    svg {
+        fill: colour(neutral-2);
+        position: absolute;
```

Comment:
Absolute gave me greater control for optical alignment with less minus margin hacky stuff. As the buttons are only ever going to be this size, it allows for less lines of css.